catalyst_voices/packages/libs/catalyst_key_derivation/lib/src/bip32_ed25519/bip32_ed25519_public_key.dart

```diff
@@ -31,6 +32,10 @@ class Bip32Ed25519XPublicKey extends Equatable {
   /// Returns the bytes of the public key.
   List<int> get bytes => _bytes.inner;
```

Comment:
Usually I do it but here I wanted to keep `toCbor()` and `toHex()` close to constructors because they usually work in a tandem, constructor converts from `something`, the method converts to `something`.

This class doesn't have `fromCbor` constructor but for others that have it keeping `fromCbor` next to `toCbor` makes sense. Here I just kep the same order even though the constructor is missing, the class follows the same concepts